src/Components/PaperEditor/TranscriptsContainer/Paragraphs/Paragraph.js

```diff
@@ -1,75 +1,125 @@
+import PropTypes from 'prop-types';
+import React from 'react';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { shortTimecode } from '@bbc/react-transcript-editor/timecodeConverter';
+import Word from './Word';
+import AnnotationOverlayTrigger from './AnnotationOverlayTrigger';
 import styles from './index.module.css';
+import cuid from 'cuid';
+import removePunctuation from '../../../../Util/remove-punctuation';
 
 /**
  *  Paragraph display preferences based on search
- *
  *  styles to separate the look of non contiguous paragraphs?
  */
+const Paragraph = (props) => {
+  const {
+    transcriptId,
+    paragraph,
+    isSearchResult,
+    handleWordClick,
+    labels,
+    handleDeleteAnnotation,
+    handleEditAnnotation,
+  } = props;
 
-class Paragraph extends Component {
+  const { speaker, words, text } = paragraph;
+  const textWithoutPunctuation = removePunctuation(text);
 
-  render() {
-    const { paragraphDisplayPreference } = this.props;
-    // const inContextSearch = paragraphDisplayPreference;
-    paragraphDisplayPreference.borderStyle = 'dashed';
-    paragraphDisplayPreference.borderWidth = '0.01em';
-    paragraphDisplayPreference.borderColor = 'lightgray';
-    paragraphDisplayPreference.padding = '0.5em';
+  const getWordWithAnnotations = (word) => {
+    const wordEl = (
+      <Word
+        transcriptId={ transcriptId }
+        speaker={ speaker }
+        key={ cuid() }
+        word={ word }
+        handleKeyPress={ handleWordClick }
+      />
+    );
 
-    // inContextSearch.display = 'block';
-    let paragraphStyle = {};
-    if (this.props.showParagraphsMatchingSearch) {
-      paragraphStyle = this.props.paragraphDisplayPreference;
+    if (word.annotation) {
+      return (
+        <AnnotationOverlayTrigger
+          key={ cuid() }
+          words={ wordEl }
+          labels={ labels }
+          annotation={ word.annotation }
+          handleDeleteAnnotation={ handleDeleteAnnotation }
+          handleEditAnnotation={ handleEditAnnotation }
+        />
+      );
     }
-    else {
-      if (!paragraphDisplayPreference.display) {
-        delete paragraphDisplayPreference.display;
-        // paragraphStyle.borderColor = 'orange';
-        // paragraphStyle.borderStyle = 'dashed';
-        // paragraphStyle.borderWidth = '0.01em';
-        paragraphStyle.borderRight = '0.1em dashed lightgrey';
-        paragraphStyle.borderLeft = '0.1em dashed lightgrey';
-      }
+
+    return wordEl;
+  };
+
+  /**
+   * find Annotation In Paragraph/words
+   */
+
+  const wordsElements = words.map((word) => getWordWithAnnotations(word));
+
+  const getDisplayPref = () => {
+    const displayPref = {
+      borderStyle: 'dashed',
+      borderWidth: '0.01em',
+      borderColor: 'lightgray',
+      padding: '0.5em'
+    };
+
+    if (isSearchResult) {
+      displayPref.borderRight = '0.1em dashed lightgrey';
+      displayPref.borderLeft = '0.1em dashed lightgrey';
```

Comment:
This adds additional thicker lines when displaying paragraphs in search results. If filtering is on (if `paragraphOnly` is set to `true` in the `TranscriptContainerTab`), the pararagraph is not displayed at all.